mousestyles/path_diversity/clean_movements.py

```diff
@@ -0,0 +1,108 @@
+import pandas as pd
+
+def clean_movements(movements, keep_index=None):
+    r"""
+    Returns a list of cleaned movement objects.
+
+    For each element of the input `movements` object, detects the existence of
+    rows which have i) same x,y-coordinates, or ii) same timestamps.
```

Comment:
I think it's important to emphasize we are detecting the existence of **adjacent** rows that have the same x, y coordinates, since it's perfectly fine to have observations with the same x, y at different time points.